test/postControllerTest.js

```diff
@@ -0,0 +1,148 @@
+/* global it, describe */
+const express = require('express');
+const request = require('supertest');
+const postController = require('../controllers/postController');
+const sinon = require('sinon');
+const postService = require('../services/postService');
+const assert = require('assert');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(express.static('../public'));
+
+app.use('', postController);
+
+describe('postController', () => {
+  beforeEach(() => {
+    sinon.stub(postService, 'getAllPosts').resolves('expected');
+    sinon.stub(postService, 'getOnePost').resolves('expected');
+    sinon.stub(postService, 'createNewPost').resolves('expected');
+  });
+
+  afterEach(() => {
+    postService.getAllPosts.restore();
+    postService.getOnePost.restore();
+    postService.createNewPost.restore();
+  });
+
+  describe('GET /', () => {
+    it('should call method "postService.getAllPosts" with no arguments', () => (
+      request(app)
+        .get('/')
+        .then(() => {
+          sinon.assert.calledWith(postService.getAllPosts);
+        })
+    ));
```

Comment:
This looks much more concise.

Not an issue: I prefer an explicit `return`, but that disagrees with your eslint config